ngraph/python/tests/__init__.py

```diff
@@ -167,8 +167,8 @@ def xfail_test(reason="Mark the test as expected to fail", strict=True):
                                       "ai.onnx.preview.training.Adagrad")
 xfail_issue_38736 = xfail_test(reason="RuntimeError: nGraph does not support the following ONNX operations:"
                                       "NegativeLogLikelihoodLoss")
-xfail_issue_43523 = xfail_test(reason="onnx.onnx_cpp2py_export.checker.ValidationError:"
-                                      " Unrecognized attribute: axes for operator ReduceSum")
+xfail_issue_45177 = xfail_test(reason="RuntimeError: axes has zero dimension which is not allowed")
+xfail_issue_45180 = xfail_test(reason="E   Unsupported dynamic op: ReduceSum")
```

Comment:
Could you provide more comprehensive description of error? Instead of E, provide the type according to a root cause (i.e. RuntimeError)

Reply:
Done, we should verify other errors descriptions too.